heareval/tasks/dcase2016_task2.py

```diff
@@ -78,17 +77,13 @@ def get_split_metadata(self, split: str) -> pd.DataFrame:
                 .replace(".txt", ".wav")
             )
             assert os.path.exists(sound_file)
-            metadata = metadata.assign(relpath=sound_file)
-
-            metadata = metadata.assign(
-                slug=lambda df: df.relpath.apply(self.slugify_file_name)
-            )
-            metadata = metadata.assign(split=lambda df: split)
             metadata = metadata.assign(
-                filename_hash=lambda df: df["slug"].apply(
-                    luigi_util.filename_to_int_hash
-                )
+                relpath=sound_file,
+                slug=lambda df: df.relpath.apply(self.slugify_file_name),
+                split=lambda df: split,
+                subsample_key=lambda df: df["slug"].apply(self.get_subsample_key),
```

Comment:
https://github.com/neuralaudio/hear2021-eval-kit/issues/113

Also the contract for these things should be documented